src/Xtools/ProjectRepository.php

```diff
@@ -206,7 +206,11 @@ public function getMetadata($projectUrl)
             return $this->metadata;
         }
 
-        $api = MediawikiApi::newFromPage($projectUrl);
+        try {
+            $api = MediawikiApi::newFromPage($projectUrl);
+        } catch (\Exception $e) {
+            return null;
```

Comment:
The doc should be updated to indicate `null` means the API call failed. I would actually instead throw a descriptive exception, because if the API failed we have bigger problems :) Also places that call `getMetadata` in Project.php will error out, such as Project::getTitle